Lib/fontTools/cffLib/__init__.py

```diff
@@ -18,47 +18,66 @@
 # mute cffLib debug messages when running ttx in verbose mode
 DEBUG = logging.DEBUG - 1
 log = logging.getLogger(__name__)
-isCFF2 = False
 
 cffHeaderFormat = """
 	major:   B
 	minor:   B
 	hdrSize: B
 """
 
-defaultMaxStack = 193
-maxStackLimit = 513
+maxStackLimit = 513 
+# maxstack operator has been deprecated. max stack is now always 513.
+
+class CFFContext(object):
```

Comment:
Instead of a context that carries nothing other than isCFF2, I would have just passed isCFF2, or maybe cffVersion down to all functions that **need it**.  Ie, probably just compile/decompile, and NOT constructors of most objects.  But I'm not sure how feasible that is.  Can be done at a later time too.